src/Events/Admin/Settings/Community_Upsell.php

```diff
@@ -9,17 +9,25 @@
 
 namespace TEC\Events\Admin\Settings;
 
-use Tribe\Events\Admin\Settings;
-use Tribe__Settings_Tab;
 use TEC\Common\Contracts\Service_Provider;
+use Tribe__Settings_Tab;
 use Tribe__Template;
+use Tribe\Events\Admin\Settings;
 
 /**
  * Class Upsell
  *
  * @since TBD
  */
 class Community_Upsell extends Service_Provider {
+	/**
```

Comment:
One extra newline, please 😄 

```suggestion

	/**
```